requirements.txt

```diff
@@ -10,9 +10,10 @@ fastavro>=1.0.0.post1
 python-snappy>=0.5.4
 charset-normalizer>=1.3.6
 psutil>=4.0.0
-scipy>=1.4.1,<1.11.0
+scipy>=1.4.1
 requests>=2.28.1
 networkx>=2.5.1
 typing-extensions>=3.10.0.2
 HLL>=2.0.3
 datasketches>=4.1.0
+packaging>=23.0
```

Comment:
let's move this to only in the graph requirements list since we don't need it elsewhere

Reply:
Nvm, there's no graph specific req file!